src/OrchardCore.Modules/OrchardCore.ContentPreview/PreviewStartupFilter.cs

```diff
@@ -2,13 +2,12 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
-using Microsoft.Extensions.Hosting;
 
 namespace OrchardCore.ContentPreview
 {
     public class PreviewStartupFilter : IStartupFilter
     {
-        public PreviewStartupFilter(IHostEnvironment hostEnvironment)
+        public PreviewStartupFilter()
```

Comment:
Yes I know but there are so many empty ctors, so because the Analyzer doesn't complain.

I also hesitated to update comments as we have no defined rules around them, so I didn't search for them, only when seeing them while updating a given file. I think we should define comment rules as starting with a space and then an uppercase, ending with a dot and so on.

Reply:
Done